libraries/postgres/test/mocha.opts

```diff
@@ -0,0 +1,3 @@
+--ui tdd
```

Comment:
iirc we use `.mocharc.yml` in the monorepo root to configure this now. This file is unnecessary.